File: playground/basic.js
// you can try this out as follows:
// 	cd stdg
// 	cargo build
// 	node playground/basic.js | target/debug/stdg[.exe]

console.log("window 400 400");
console.log("title A Rectangle");

while (true) {
	console.log("color 255 255 255");
	console.log("clear");
	console.log("color 255 0 0");
	console.log("rect 50 50 100 100");
	console.log("present");
	console.log("handle");
}

File: playground/basic.go
// you can try this out as follows:
// 	cd stdg
// 	cargo build
// 	go run playground/basic.go | target/debug/stdg[.exe]

package main

import "fmt"

func main() {
	fmt.Println("window 400 400")
	fmt.Println("title A Rectangle")

	for {
		fmt.Println("color 255 255 255")
		fmt.Println("clear")
		fmt.Println("color 255 0 0")
		fmt.Println("rect 50 50 100 100")
		fmt.Println("present")
		fmt.Println("handle")
	}
}